.github/ISSUE_TEMPLATE.md

```diff
@@ -1,39 +0,0 @@
-### Prerequisites
-
-The issue tracker is used to report bugs and request new features, **NOT** to ask questions.
-
-Questions should be posted in [Discussions](https://github.com/IronLanguages/ironpython3/discussions/categories/q-a) or to the users mailing list which can be accessed at 
-https://ironpython.groups.io/g/users. 
-
-* [ ] Are you running the latest version?
-* [ ] Are you reporting to the correct repository?
-* [ ] Did you perform a cursory search?
-
-### Description
-
-[Description of the bug or feature]
-
-### Steps to Reproduce
-
-1. [First Step]
-2. [Second Step]
-3. [and so on...]
-
-**Expected behavior:**
-
-[What you expected to happen]
-
-**Actual behavior:**
-
-[What actually happened]
-
-### Version Information
```

Comment:
Hmm, not sure what the issue templates changes will look like in practice, but we want the information from the Version Information section.

Reply:
I have similar issue templates in a few of my repos, [here](https://github.com/RiversideValley/Graphite/issues/new/choose)'s a repo that you can see what they look like in practice. Feel free to make any issues on that repo to try them out, I don't mind.

Reply:
As for the version information, I'll had that soon.